ext/scheduler/airflow/airflow.go

```diff
@@ -286,6 +287,37 @@ func (s *Scheduler) GetJobRuns(ctx context.Context, tnnt tenant.Tenant, jobQuery
 	return getJobRuns(dagRunList, jobCron)
 }
 
+func (s *Scheduler) UpdateJobState(ctx context.Context, tnnt tenant.Tenant, jobName, state string) error {
```

Comment:
we can pass the job name as  `job.Name` instead of string

Reply:
okay